samples/nextjs/src/pages_examples/[[...path]].SSR.tsx

```diff
@@ -58,20 +60,21 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     .fetchLayoutData(path, props.locale, req, res)
     .catch((error: AxiosError) => {
       // Let 404s (invalid path) through
-      if (error.response?.status === 404) return null;
+      if (error.response?.status === 404) return error.response.data;
```

Comment:
What is the `error.response.data` in this context / what do we gain over the null return? Also, the SSG version uses `AxiosError<LayoutServiceData>` for the error type (not sure if that's at play here).

Reply:
@ambrauer `error.response.data` it's `{ sitecore: { context } }` data, we need this data inside `NotFound` component

Reply:
Ah, of course.